src/pages/deployer/index.tsx

```diff
@@ -111,24 +108,68 @@ function DeployerPage() {
 
   return (
     <Screen>
-      <Navbar />
       <TxLoader open={isLoading}>
         <StyledTxLoaderContent>
           <Typography>Deploying...</Typography>
         </StyledTxLoaderContent>
       </TxLoader>
       <ScreenContent removeBackground>
         <Fade in>
-          <StyledContainer>
-            <StyledLeft>
-              <SearchInput />
-              <Description />
-            </StyledLeft>
-            <StyledRight>
-              <SectionLabel>Create your own new Jetton</SectionLabel>
-              <Form submitText="Deploy" onSubmit={deployContract} inputs={formSpec} />
-            </StyledRight>
-          </StyledContainer>
+          <Box>
+            <Box mb={3} mt={0}>
+              <Typography
+                sx={{
+                  fontWeight: 800,
+                  color: "#161C28",
+                  fontSize: 48,
+                  marginTop: 5,
+                  [theme.breakpoints.down("md")]: {
+                    fontSize: 30,
+                    textAlign: "center",
+                  },
+                  [theme.breakpoints.down("sm")]: {
+                    fontSize: 24,
+                  },
+                }}
+                variant="h5">
+                Mint your token
+              </Typography>
+            </Box>
+            <Box
+              sx={{
+                display: "flex",
+                alignItems: "stretch",
+                gap: 5,
+                [theme.breakpoints.down("lg")]: {
+                  flexDirection: "column",
+                },
+              }}>
+              <Box
+                sx={{
+                  flex: 5,
+                  background: "#FFFFFF",
+                  border: "0.5px solid rgba(114, 138, 150, 0.24)",
+                  boxShadow: "0px 2px 16px rgba(114, 138, 150, 0.08)",
+                  borderRadius: "24px",
+                  padding: 3,
+                }}>
+                <Typography
+                  variant="h5"
+                  mb={3}
+                  sx={{
+                    color: "#161C28",
+                    fontSize: 20,
+                    fontWeight: 800,
+                  }}>
```

Comment:
is it possible to extract styling out? it makes the component harder to read

Reply:
done